ocs_ci/deployment/deployment.py

```diff
@@ -134,20 +131,74 @@ def label_and_taint_nodes(self):
             )
             _ocp.exec_oc_cmd(command=taint_cmd)
 
+    def get_olm_manifest(self):
+        """
+        This method prepare manifest for deploy OCS operator.
+
+        Returns:
+            str: Path to olm deploy manifest
+
+        """
+        image = config.DEPLOYMENT.get('ocs_operator_image', '')
+        image_and_tag = image.split(':')
+        image = image_and_tag[0]
+        image_tag = image_and_tag[1] if len(image_and_tag) == 2 else None
+        if not image_tag and config.REPORTING.get("us_ds") == 'DS':
```

Comment:
Once we are using `us_ds` parameter not only for reporting, we should probably move it to different section (e.g. `DEPLOYMENT`) and change the [description](https://github.com/red-hat-storage/ocs-ci/blob/b1ab32e4c2c2324ca6d36f40063220b4b2b8c6ce/ocs_ci/framework/conf/default_config.yaml#L57). It will require change also on the other places, where it is used. WDYT?

Reply:
Yep I agree but will do it in different PR as it affect more stuff not only in our CI but also engineering python script for configuration. Opened issue for it here and will resolve this conversation.  https://github.com/red-hat-storage/ocs-ci/issues/764